docs/glossary.rst

```diff
@@ -23,6 +23,13 @@ Glossary of Terms
       snapshot of the system's files prior to and after a suspected incident.
       It is maintained by Rami Lehti and Pablo Virolainen.
 
+   APL
+   Automatic Parameter Lookup
+      A method where Puppet automatically looks up class parameters in
+      :term:`Hiera` using the fully-qualified name of the parameter.
+
+      See: `Looking up data with Hiera <https://puppet.com/docs/puppet/5.3/hiera_automatic.html#puppet-lookup>`__
```

Comment:
I habitually replace the version-number in Puppet links with `latest`.  Of course, this works best when you have an automatic link-checker in your pipeline.

Reply:
@pillarsdotnet I think there was a problem with this one that I had to file a bug about actually. We usually do that as well and we do have an automatic link-checker in the pipeline.

Reply:
Hmm...looks like they fixed it, will update